### ISSUE TITLE
Error reported in build TensorRT inference engine for the bert model

### ISSUE THREAD

**Issue by scxiao** [NVIDIA/TensorRT#1760]
## Description

From TensorRT docker image 21.07, different errors are reported in building the Inference engine for the BERT Squad model.

## Environment

**TensorRT Version**: 
**NVIDIA GPU**: 
**NVIDIA Driver Version**: 
**CUDA Version**: 
**CUDNN Version**: 
**Operating System**: 
**Python Version (if applicable)**: 
**Tensorflow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if so, version)**: 

## Relevant Files

Link to the model: https://zenodo.org/record/3733910/files/model.onnx, the command line is: 

```
trtexec --onnx=model.onnx --fp16 --shapes=input_ids:1x384,input_mask:1x384,segment_ids:1x384 
```

The error info is:
trt version 21.12-py3

[01/14/2022-20:12:41] [W] [TRT] Output type must be INT32 for shape outputs
[01/14/2022-20:12:41] [W] [TRT] Output type must be INT32 for shape outputs
[01/14/2022-20:12:41] [I] Finish parsing network model
[01/14/2022-20:12:45] [I] [TRT] [MemUsageChange] Init cuBLAS/cuBLASLt: CPU +375, GPU +176, now: CPU 3215, GPU 697 (MiB)
[01/14/2022-20:12:45] [I] [TRT] [MemUsageChange] Init cuDNN: CPU +116, GPU +52, now: CPU 3331, GPU 749 (MiB)
[01/14/2022-20:12:45] [I] [TRT] Local timing cache in use. Profiling results in this builder pass will not be stored.
[01/14/2022-20:13:20] [I] [TRT] Some tactics do not have sufficient workspace memory to run. Increasing workspace size may increase performance, please check verbose output.
[01/14/2022-20:13:46] [E] Error[10]: [optimizer.cpp::computeCosts::2011] Error Code 10: Internal Error (Could not find any implementation for node {ForeignNode[bert.embeddings.word_embeddings.weight...node_of_output_start_logits]}.)
[01/14/2022-20:13:46] [E] Error[2]: [builder.cpp::buildSerializedNetwork::609] Error Code 2: Internal Error (Assertion enginePtr != nullptr failed. )
[01/14/2022-20:13:46] [E] Engine could not be created from network
[01/14/2022-20:13:46] [E] Building engine failed
[01/14/2022-20:13:46] [E] Failed to create engine from model.
[01/14/2022-20:13:46] [E] Engine set up failed

trt version 21.09-py3
[01/14/2022-20:21:36] [E] Error[10]: [optimizer.cpp::computeCosts::1855] Error Code 10: Internal Error (Could not find any implementation for node {ForeignNode[440 + (Unnamed Layer* 82) [Shuffle]...node_of_output_start_logits]}.)
[01/14/2022-20:21:36] [E] Error[2]: [builder.cpp::buildSerializedNetwork::417] Error Code 2: Internal Error (Assertion enginePtr != nullptr failed.)
Segmentation fault (core dumped)

trt version 21.07-py3
[01/14/2022-20:28:14] [I] Finish parsing network model
[01/14/2022-20:28:14] [I] [TRT] [MemUsageChange] Init CUDA: CPU +0, GPU +0, now: CPU 2697, GPU 481 (MiB)
[01/14/2022-20:28:14] [I] [TRT] [MemUsageSnapshot] Builder begin: CPU 2697 MiB, GPU 481 MiB
[01/14/2022-20:28:18] [I] [TRT] [MemUsageChange] Init cuBLAS/cuBLASLt: CPU +350, GPU +160, now: CPU 3049, GPU 641 (MiB)
[01/14/2022-20:28:19] [I] [TRT] [MemUsageChange] Init cuDNN: CPU +159, GPU +164, now: CPU 3208, GPU 805 (MiB)
[01/14/2022-20:28:19] [W] [TRT] Detected invalid timing cache, setup a local cache instead
[01/14/2022-20:28:30] [W] [TRT] Skipping tactic 0 due to oom error on requested size of 34604544 detected for tactic 0.
[01/14/2022-20:28:42] [W] [TRT] Skipping tactic 0 due to oom error on requested size of 17302272 detected for tactic 0.
[01/14/2022-20:28:42] [I] [TRT] [MemUsageChange] Init cuBLAS/cuBLASLt: CPU +0, GPU +0, now: CPU 3208, GPU 787 (MiB)
[01/14/2022-20:28:42] [E] Error[10]: [optimizer.cpp::computeCosts::1855] Error Code 10: Internal Error (Could not find any implementation for node {ForeignNode[440 + (Unnamed Layer* 82) [Shuffle]...node_of_output_start_logits]}.)
[01/14/2022-20:28:42] [E] Error[2]: [builder.cpp::buildSerializedNetwork::417] Error Code 2: Internal Error (Assertion enginePtr != nullptr failed.)
Segmentation fault (core dumped)

**Comment by scxiao** [comment]
This error is caused by the default workspace value. By setting a flag "--workspace=2048", it works. So close this issue.
